modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on a public DNS zone on Infoblox
+
+You can create DNS records on a public DNS zone on Infoblox by using the Red Hat External DNS Operator.
+
+.Prerequisites
+
+* You have an access to OpenShift CLI (oc).
+* You have an access to Infoblox UI.
+
+.Procedure
+
+. Create a `secret` object with Infoblox credentials by running the following command:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=<INFOBLOX_USERNAME> --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=<INFOBLOX_PASSWORD>
```

Comment:
Guidelines aren't explicit about this, but every example is lowercase so I think that is the intended style:

```suggestion
$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=<infoblox_username> --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=<infoblox_password>
```

Reply:
Addressed